x/staking/types/params.go

```diff
@@ -28,7 +28,7 @@ const (
 
 	// DefaultHistorical entries is 0 since it must only be non-zero for
 	// IBC connected chains
-	DefaultHistoricalEntries uint32 = 0
+	DefaultHistoricalEntries uint32 = 1000
```

Comment:
Any rationale behind this value?

Reply:
/cc @AdityaSripal

Reply:
This will govern how low-latency IBC handshakes must be in order to complete in time. I think `1000` is fine, it's probably even fine to use something much lower, like `100`.

Reply:
I see. Are historical entries persisted in state? If so, we should opt for a lower bound instead.

Reply:
after block 100 the new ones start overriding the old ones so there's always at most 100